lib/errorSerialization.js

```diff
@@ -0,0 +1,18 @@
+var serializeError = require('serializerr');
+var deserializeError = require('deserialize-error');
+
+exports.serializeError = function (error) {
```

Comment:
for consistency with the other serializer, can we have those named "toJSON" and "fromJSON" respectively?